### PR TITLE
New solution Export Crashlytics logs to Datadog.

### PR DESCRIPTION
**Pull request by anantdamle**
New solution Export Crashlytics logs to Datadog.

### CODE REVIEW

**Comment by anantdamle** [comment]
@ToddKopriva I checked the description against the provided REGEX and it passes. is there something I'm missing?

File: tutorials/crashlytics-logs-to-datadog/index.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    --stagingLocation="gs://${GCS_BUCKET}/staging" \
+    --tempLocation="gs://${GCS_BUCKET}/bqtemp" \
+    --datadogApiKey="${DATADOG_API_KEY}" \
+    --sourceBigQueryTableId="${CRASHLYTICS_BIGQUERY_TABLE}"
+    
+You can monitor the Dataflow job on Cloud Console. The pipeline DAG looks as follows:
+
+![Pipeline DAG](pipeline_dag.png)
+
+### Create Dataflow flex template
```
Comment (ktchana) [inline tutorials/crashlytics-logs-to-datadog/index.md:205]: As the previous step already contains the command to launch the pipeline, this step may look a bit confusing to some audiences. I would suggest marking this part as optional and also spell out the reason why would one choose to create a flex template over running it directly (like for launching from env without JRE?)

**Review by ktchana — COMMENTED** [review]
One minor comment on flex template. Otherwise the tutorial looks good to me.

File: tutorials/crashlytics-logs-to-datadog/index.md (hunk 1 above)
Comment (anantdamle) [reply tutorials/crashlytics-logs-to-datadog/index.md:209]: Thanks @ktchana: Done. in the latest commit.

**Review by ktchana — APPROVED** [review]
LGTM

**Review by ToddKopriva — APPROVED** [review]

**Pull request merged**